resources/assets/js/markdown-editor/plugins/underline.js

```diff
@@ -0,0 +1,127 @@
+import { createButtonWithIcon } from "../utils/utils";
+
+/**
+ * Adds the menu button
+ * @param {object} editor - Editor instance
+ * @param {int} MenuIndex - index inside the menu
+ * @param {object} svgIcon - Svg template
+ * @ignore
+ */
+const addToolbarButton = (editor, menuIndex, svgIcon) => {
+    const toolbar = editor.getUI().getToolbar();
+
+    toolbar.insertItem(menuIndex, {
+        type: "button",
+        options: {
+            el: createButtonWithIcon(svgIcon),
+            name: "underline",
+            className: "tui-underline",
+            command: "underline",
+            tooltip: "underline",
+        },
+    });
+
+    return toolbar.getItem(menuIndex);
+};
+
+const isUnderlineSelected = (editor) => {
+    const selectedText = editor.getSelectedText();
+
+    if (editor.currentMode === "wysiwyg") {
+        const parentNode = editor.getRange().startContainer.parentNode;
+        return parentNode.tagName === "U" || parentNode.closest("U") !== null;
+    }
+
+    if (selectedText.startsWith("<ins>") && selectedText.endsWith("</ins>")) {
```

Comment:
I suppose we can add a style rule to have `ins` always be underlined in places where we show our markdown.

Reply:
@ItsANameToo yes also we may need to update the markdown parser to allow that specific tag currently is being filtered

Reply:
I added a card to clickup for that